src/cozmo/robot.py

```diff
@@ -1273,7 +1273,13 @@ def perform_off_charger(self):
         perform the code wrapped by the context and then move the robot back onto the
         charger after the wrapped code completes.
 
-        For example::
+        Synchronous example::
+
+            with robot.perform_off_charger():
+                action = robot.say_text("Hello")
+                action.wait_for_completed()
+
+        Asynchronous example::
 
             async with robot.perform_off_charger():
                 action = robot.say_text("Hello")
```

Comment:
The differences are both the "async with" and the "await". For copying and pasting code, I think the full example could be handy.